examples/posenet/src/PoseNet.js

```diff
@@ -1,5 +1,5 @@
 class PoseNet{
-  constructor(modelArch, backend, version, outputStride, inputShape, type, cacheMap) {
+  constructor(modelArch, version, outputStride, inputShape, type, cacheMap, backend) {
```

Comment:
What's the reason to change the signature? Did we change every reference of this API?

Reply:
Changing this signature for invoking PoseNet adapt for Benchmark test, let backend as default function parameter, if backend had no value, undefined is passed.

Reply:
sounds good.